src/main/webapp/creditCardTransactions.jsp

```diff
@@ -8,10 +8,10 @@
 </head>
 <body style='background-color:orange;'>
 
-<h2 input style="margin-left: 30em";>Authorize Credit Card Transaction </h2>
+<h2 input style="margin-left: 30em";>Authorize Credit Card Transaction </h2> 
 
 <form action="AuthorizeCCTransactionsServlet" method = "post">
-<label for="nameOnCreditCard">Name:</label>  <input  style="margin-left: 3.7em"; type = "text" id = "nameOnCreditCard" name = "nameOnCreditCard" placeholder="Dipanjan Sengupta"> <br> <br>
+<label for="nameOnCreditCard">Name:</label>  <input  style="margin-left: 3.7em"; type = "text" id = "nameOnCreditCard" name = "nameOnCreditCard" placeholder="Dipanjan Sengupta "> <br> <br>
 <label for="cardNumber">Card Number:</label>  <input type = "text" id = "cardNumber" name = "cardNumber" placeholder="16 digit number"> <br> <br>
 <label for="cvvCode">CVV2/CVC2:</label>  <input  style="margin-left: .1em"; type = "text" id = "cvvCode" name = "cvvCode" placeholder="3 digit number"> <br> <br>
 <label for="amount">Amount:</label>  <input  style="margin-left: 2.6em"; type = "text" id = "amount" name = "amount" placeholder="Value less than 100000"> <br> <br>
```

Comment:
- do not display unable to connect to database. make the message less verbose
- placeholder should have meaningful text, not exact value

Reply:
do not display unable to connect to database. make the message less verbose
**_"Bank Unreachable , check Internet Connection/try later"_**
placeholder should have meaningful text, not exact value
**_"Eta Edureka'r jonno korechi"_**